### PR TITLE
feat: use hook context

### PR DESCRIPTION
**Pull request by turrs**
create use hook context

### CODE REVIEW

File: src/main.jsx [hunk 1]
```diff
@@ -3,9 +3,31 @@ import ReactDOM from "react-dom/client";
 import App from "./App";
 import "./index.css";
 import "antd/dist/antd.css";
-
+import {
```
Comment (argorfc) [inline src/main.jsx]: Please use Global Providers

**Review by argorfc — CHANGES_REQUESTED** [review]
Read the comments!

File: src/main.jsx (hunk 1 above)
Comment (turrs) [reply src/main.jsx]: already fix use global providers here 2c3de69d20d22a427909046f0a945e8f681790b3

**Review by argorfc — APPROVED** [review]
Approved.
Ready to merge.

**Pull request merged**